### DISCUSSION TITLE
Error building Fox nerf: VK_NVX_binary_import not supported...

### DISCUSSION THREAD

**Discussion by offlinegeek** [NVlabs/instant-ngp#631] [Q&A]
Hi, thanks for looking at this problem.

./build/testbed --scene data/nerf/fox throws three errors:

ERROR    Vulkan: loader_validate_device_extensions: Device extension VK_NVX_binary_import not supported by selected physical device or enabled layers.

ERROR    Vulkan: vkCreateDevice: Failed to validate extensions in list

ERROR    Uncaught exception: /home/test/instant-ngp/dependencies/tiny-cuda-nn/include/tiny-cuda-nn/gpu_memory.h:556 cuMemCreate(&m_handles.back(), n_bytes_to_allocate, &prop, 0) failed with error CUDA_ERROR_OUT_OF_MEMORY

Any suggestions please? Vulkan API powered games such as DOTA2 work without problem.  The graphics card is quite old but the latest NVIDIA drivers apparently still support it. The first time I used CMake to build the project I did not have the Vulkan SDK installed so I rm -rd the build folder and remade after installing the Vulkan SDk. 

This is very much the "hello world" stage on the nerf learning curve for me!  

The system specs are:

15.6GB Memory
GTX970 
Ubuntu 20.04
NVIDIA-SMI 515.43.04    Driver Version: 515.43.04    
CUDA Version: 11.7 
Vulkan Instance Version: 1.3.211

**Comment by Tom94** [comment]
Hi there, the Vulkan errors don't really matter for being able to run __instant-ngp__. If Vulkan fails to initialize DLSS will not work, but that's about it. I've just pushed a commit that reports Vulkan problems as warnings rather than errors (to avoid this sort of ambiguity).

The last error message is the relevant one: you're running out of GPU memory -- the 970 only has 4GB of RAM. To fix this, I'm afraid the only way (short of upgrading the GPU) is to reduce the dataset size.

**Comment by offlinegeek** [comment]
Hi, thank you for your quick reply. I will try with a reduced dataset for now and budget for a new graphics card!
